manifests/base/manager/kustomization.yaml

```
@@ -1,2 +1,12 @@
 resources:
   - manager.yaml
+
+# ConfigMap generator for controller manager configuration
+configMapGenerator:
+  - name: manager-config
```

Shall we just call it `kubeflow-trainer-config`, since in the future we can use this API for some other control plane components, not manager (e.g. Trainer UI). WDYT @kubeflow/kubeflow-trainer-team ?
```suggestion
  - name: kubeflow-trainer-config
```